extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/RemoveDeploymentTriggerDecorator.java

```diff
@@ -20,12 +20,12 @@ public void andThenVisit(DeploymentConfigSpecFluent<?> deploymentConfigSpec, Obj
     }
 
     @Override
-    public Class<? extends Decorator>[] after() {
-        return new Class[] { ApplyDeploymentTriggerDecorator.class };
+    public Class<? extends Decorator>[] before() {
+        return new Class[] { ChangeDeploymentTriggerDecorator.class };
     }
 
     @Override
-    public Class<? extends Decorator>[] before() {
-        return new Class[] { ChangeDeploymentTriggerDecorator.class };
+    public Class<? extends Decorator>[] after() {
+        return new Class[] { ApplyDeploymentTriggerDecorator.class };
```

Comment:
Do we need this change?

Reply:
Yes we do. Pretty much all remove decorators are meant to be processes after the apply ones

Reply:
I am wrong, we don't need this change, however I think that it's ok to have.

Reply:
I agree that semantically it's the same, but I would avoid making changes that are unrelated to the issue itself (which is about configuring the port of HTTP probes).